gravitee-apim-portal-webui-next/src/app/app.component.ts

```diff
@@ -36,9 +37,11 @@ export class AppComponent {
   logo = inject(ThemeService).logo;
   favicon = inject(ThemeService).favicon;
   siteTitle: string;
+  customLinks: PortalMenuLink[];
```

Comment:
feel free to inject it like the others above --> `inject(PortalMenuLinksService).links`